addons/website_sale/views/templates.xml

```diff
@@ -800,15 +800,17 @@
                     <t t-foreach="entries" t-as="c" t-if="c.image_128" t-set="atLeastOneImage" t-value="True"/>
                     <t t-if="category.parent_id" t-set="backUrl" t-value="keep('/shop/category/' + slug(category.parent_id), category=0)"/>
                     <t t-else="" t-set="backUrl" t-value="'/shop'"/>
-
+                    <!-- TODO VCR don't show sibling category on last nested level -->
                     <li
+                        t-if="c.id != category.id"
```

Comment:
Since we're not able to remove the sibling category,  we have keep the current `master` behaviour.
Otherwise it will look like the sibling it's a subcategory 

| | |
|--------|--------|
| parent - `master` | sub - `master` |
| ![image](https://github.com/user-attachments/assets/93ac6100-f433-4686-9059-38796f516881) | ![image](https://github.com/user-attachments/assets/80df0aa8-129f-4ca4-8c61-f1cc22c5cbe4) |
| parent - this PR | sub - this PR |
| ![image](https://github.com/user-attachments/assets/79fa9fd3-8042-4ec4-bedc-7be264e77066) | ![image](https://github.com/user-attachments/assets/5653c5e5-652d-4608-b3e9-e4a0157112fd) |